projects/canopy/src/lib/progress-indicator/progress-indicator.component.html

```diff
@@ -0,0 +1,12 @@
+<lg-progress-journey
+  [max]="max"
+  [value]="value"
+  [showAsPercentage]="showAsPercentage"
+  [stepsPrefix]="stepsPrefix"
+>
+  <ng-content />
+</lg-progress-journey>
+
+<lg-progress-bar *ngIf="showProgressBar" [max]="max" [value]="value" />
```

Comment:
It would  great to add some tests to ensure that the `max` and `value` inputs have been set correctly, as if someone were to come along and change `[max]="max"` to say `[max]="'100'"` it would break the implementation.